From 81384bed832ab9d4da89f1e9aba36ecce0d80055 Mon Sep 17 00:00:00 2001
From: "Yue, VincentX" <vincentx.yue@intel.com>
Date: Fri, 22 Dec 2017 15:10:27 +0800
Subject: [PATCH 2/4] Allow system_app read system_server_tmpfs

The BatteryStatsService.getStatisticsStream creat MemoryFile
for save battery information, relax the system_app to read.

Change-Id: Ic33388343712a9b360224357f19488e61f6859b9
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-54566
Signed-off-by: Yue, VincentX <vincentx.yue@intel.com>
---
 private/system_app.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/private/system_app.te b/private/system_app.te
index 4741479..f4ead15 100644
--- a/private/system_app.te
+++ b/private/system_app.te
@@ -61,6 +61,9 @@ allow system_app servicemanager:service_manager list;
 # TODO: scope this down? Too broad?
 allow system_app { service_manager_type -netd_service -dumpstate_service -installd_service -virtual_touchpad_service -vr_hwc_service }:service_manager find;
 
+# Allow system app to read sharedMemory system service created
+allow system_app system_server_tmpfs:file read;
+
 allow system_app keystore:keystore_key {
     get_state
     get
-- 
1.9.1

